### PR TITLE
feat(unify): Update docs links

### PR DESCRIPTION
**Pull request by mapsandapps**
- Closes [UNIFY-1079]

Note: I skipped the error message links mentioned in the ticket. I didn't see anything coming back in the current GraphQL query that would allow me to know what links to use easily. We broke that one out to UNIFY-1188.

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

[UNIFY-1079]: https://cypress-io.atlassian.net/browse/UNIFY-1079?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jaffrepaul** [comment]
Also, do we have a sense of how we can prioritize the [error message ticket](https://cypress-io.atlassian.net/browse/UNIFY-1188)? I'm not sure how much work this is.

**Comment by mapsandapps** [comment]
Someone else will have to speak to the error docs ticket. I don't think I'm the right person to work that one. Like I said above, it doesn't seem like anything that comes back in the graphql query (or what's available from what i could see in graphliql) would make it easy to determine what the link would be. I can probably ask in the CT team meeting today.

**Review by jaffrepaul — APPROVED** [review]
LGTM 🎉

File: packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue [hunk 1]
```diff
@@ -4,6 +4,7 @@
     variant="bare"
     :title="title"
     :model-value="show"
+    help-link=""
```
Comment (elevatebart) [inline packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue:7]: I am not really comfortable with this.
An empty sting does not mean "hide the link" in my book. It means show the link with an empty target.

I would prefer a quick refactoring of the modal to make is so to this empty prop.

**Review by elevatebart — CHANGES_REQUESTED** [review]
It feels like a lot of hardcoding of `on.cypress.io` with the possibility of mistake. could we make it a simpler usage of that prop:
- props is not mentioned -> hide link
- prop is mentioned -> add it to the on.cypress.io root
- prop is empty -> show only the on.cypress.io

File: packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue (hunk 1 above)
Comment (mapsandapps) [reply packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue:7]: If you're comfortable _reviewing_ that change, I'm happy to make it. I thought about that but a) it's probably not my place to make that decision and we already had this pattern and were using it and b) I was worried I might break something and not see it. It's very hard to find all these modal states in the app or in the code to confirm that they all have the correct behavior.

Comment (elevatebart) [reply packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue:7]: Can I suggest we create a ticket to refactor the modal help links? 
This way the technical debt accumulated here is tracked somewhere.
As soon I see the ticket, it is green to me.

Comment (mapsandapps) [reply packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue:7]: [Done](https://cypress-io.atlassian.net/browse/UNIFY-1197)!

**Review by tbiethman — APPROVED** [review]

**Review by elevatebart — APPROVED** [review]

**Comment by elevatebart** [comment]
LGTM!

**Pull request merged**